gear-test/Cargo.toml

```diff
@@ -17,7 +17,7 @@ hex = "0.4.2"
 serde = { version = "*", features = ["derive"] }
 serde_json = "*"
 log = "0.4.14"
-parity-scale-codec = { version = "*", features = ["derive"], default-features = false }
+parity-scale-codec = { version = "2.2.0", features = ["derive"], default-features = false }
```

Comment:
The last version in 2.x branch is 2.3.1

Reply:
We cannot use 2.3.1 because it does not support String encode

Reply:
> We cannot use 2.3.1 because it does not support String encode

Even with "full" feature enabled?🧐

Reply:
Yep) thank you - full feature fix this